bdns/dns.go

```diff
@@ -150,11 +151,17 @@ var (
 	}
 )
 
+type ResolverAddr string
+
+func (r ResolverAddr) String() string {
+	return string(r)
+}
+
```

Comment:
I wanted the additional type so that returns from `//bdns` all the way back to `//va` are clear what the field should be. Like, it gets burdensome when I have to keep a mental model of, "Ah yes, this string is actually a dns resolver address" across however many functions and methods. Removing the stringer seems fine. I can always cast to string when necessary.

Reply:
Okay, in that case, you should consider using `string` internal to the package and only cast to a ResolverAddr when you are returning from an exported method or function. This will make collecting up results and joining them a little easier.